pyaedt/maxwell.py

```diff
@@ -150,6 +159,11 @@ def assign_matrix(
     ):
         """Assign a matrix to the selection.
 
+        Matrix assignment can be calculated based upon the solver type.
+        For 2D/3D solvers the available solution types are: ``Magnetostatic``,
+        ``Electrostatic``, ``Eddy Current``, ``DC Conduction`` and ``AC Conduction``.
```

Comment:
```suggestion
        Matrix assignment can be calculated based on the solver type.
        For 2D and 3D solvers, the available solution types are:
        
        - ``Magnetostatic``
        - ``Electrostatic``
        - ``Eddy Current``
        - ``DC Conduction``
        -  ``AC Conduction``
```
Should each list item also be in quotes because they are text strings?